src/main/java/net/mcreator/ui/minecraft/states/block/JBlockPropertiesStatesList.java

```diff
@@ -91,13 +86,20 @@ public JBlockPropertiesStatesList(MCreator mcreator, IHelpContext gui,
 		scrollProperties.getVerticalScrollBar().setUnitIncrement(15);
 		scrollProperties.setOpaque(false);
 		scrollProperties.getViewport().setOpaque(false);
-		JPanel left = new JPanel(new BorderLayout());
-		left.setOpaque(false);
-		left.add("North", PanelUtils.join(FlowLayout.LEFT, 0, 5, addProperty, new JEmptyBox(5, 5),
+
+		JPanel mainContent = new JPanel(new BorderLayout());

```

Comment:
The thing you removed in this commit was ready to also host states list later, does this look differently now?

Reply:
For someone reading code in master (which should be stable and finalized), it would be confusing to read left for a component that is not left.

There is still likelihood of needing two pages for properties and states, especially once we add things like bounding box definitions to the mix, or UI will be way too cramped